### DISCUSSION TITLE
Is it safe to reuse a list of ParserOption's returned from various jwt.With... functions?

### DISCUSSION THREAD

**Discussion by jordaniversen** [lestrrat-go/jwx#663] [General]
Is it safe to do for example something like this:

```
options := []jwt.ParseOption{jwt.WithPedantic(true)}
token, err := jwt.ParseString(rawJwt, options...)
```

or is there something that would make this unsafe or not recommended?

**Comment by lestrrat** [comment]
I think the answer is: it depends. There are some options that passed values to be modified -- these are not safe. Others they just pass a flag, which are safe.
